cypress/e2e/pages/onDemandAudio/index.cy.js

```diff
@@ -1,32 +1,487 @@
+    service: 'hausa',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/hindi/podcasts/p0552909', // Podcast Brand
+    service: 'hindi',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/hindi/podcasts/p0552909/p09kjqjm', // Podcast Episode
+    service: 'hindi',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/indonesia/bbc_indonesian_radio/programmes/w13xtt0s', // On Demand Brand
+    service: 'indonesia',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/indonesia/bbc_indonesian_radio/w172xybnvm6718v', // On Demand Episode
+    service: 'indonesia',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/indonesia/podcasts/p02pc9v6', // Podcast Brand
+    service: 'indonesia',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/indonesia/podcasts/p02pc9v6/p09l1mhb', // Podcast Episode
+    service: 'indonesia',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/korean/bbc_korean_radio/programmes/w13xttll', // On Demand Brand
+    service: 'korean',
+    runforEnv: 'live',
+    tests,
+  },
+  {
+    path: '/korean/bbc_korean_radio/programmes/w13xttlm', // On Demand Brand
+    service: 'korean',
+    runforEnv: 'test',
+    tests,
+  },
+  {
+    path: '/korean/bbc_korean_radio/w3ct1vk5', // On Demand Episode
+    service: 'korean',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/kyrgyz/bbc_kyrgyz_radio/programmes/p0340xth', // On Demand Brand
+    service: 'kyrgyz',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/kyrgyz/bbc_kyrgyz_radio/w3ct1vw9', // On Demand Episode
+    service: 'kyrgyz',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/marathi/podcasts/p09431p4', // Podcast Brand
+    service: 'marathi',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/marathi/podcasts/p09431p4/p0f1vp21', // Podcast Episode
+    service: 'marathi',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/nepali/bbc_nepali_radio/programmes/p0340xzt', // On Demand Brand
+    service: 'nepali',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/nepali/bbc_nepali_radio/w172xzcfvptk838', // On Demand Episode
+    service: 'nepali',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/nepali/podcasts/p02pc9w3/p09j0dm5', // Podcast Episode
+    service: 'nepali',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/nepali/podcasts/p02pc9w3', // Podcast Brand
+    service: 'nepali',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/pashto/bbc_pashto_radio/programmes/p0340yr4', // On Demand Brand
+    service: 'pashto',
+    runforEnv: 'live',
+    tests,
+  },
+  {
+    path: '/pashto/bbc_pashto_radio/programmes/p0340ysc', // On Demand Brand
+    service: 'pashto',
+    runforEnv: 'test',
+    tests,
+  },
```

Comment:
Because test points to live data, we can pick one path and combine them into a single entry, with runforEnv: ['test', 'live']